README.md

```diff
@@ -113,9 +115,26 @@ docker exec -it <CONTAINER NAME> /bin/bash
 $ root@ec562fe2172b:/godwoken-web3# pm2 monit
 ```
 
-Http url: http://your-url/
+#### URLs
+
+```sh
+# Http 
+http://example_web3_rpc_url
+
+# WebSocket
+ws://example_web3_rpc_url/ws
+```
 
-WebSocket url: ws://your-url/ws
+With instant-finality feature turn on:
+
+```sh
+# Http 
+http://example_web3_rpc_url?instant-finality-hack=true
+http://example_web3_rpc_url/instant-finality-hack
+
+# WebSocket
+ws://example_web3_rpc_url/ws?instant-finality-hack=true
+```
```

Comment:
Most of users don't have to care about instant-finality.
How about just only guiding to docs/addtional-feature.md#Instant%20Finality ?

Reply:
I agree but prefer to move it later, we also have to add it in godwoken-info, so let's make people notice these changes first